solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -59,21 +62,27 @@ key_name            = "devops-key"
 terraform validate && terraform apply -var-file user.tfvars --auto-approve
 ```
 
+**Note:** AWS instance public ipaddress of N-nodes can be selected from the terraform script execution output.
```

Comment:
You can use something like this https://github.com/Seagate/cortx-re/blob/aa6d5110f49f7c9e1657870d9ef991cd81ec96b7/jenkins/deployment/community/cortx-community-deploy.groovy#L79 to print IP details

Reply:
@shailesh-vaidya, Its already in the script as per output,
Apply complete! Resources: 63 added, 0 changed, 0 destroyed.

Outputs:
aws_instance_public_ip_addr = [
  "52.66.139.x",
  "3.111.187.x",
  "13.233.26.x",
]